### ISSUE TITLE
Include formula of Dynamic menu item based on user.id or custom field user.category

### ISSUE THREAD

**Issue by castoutside** [saltcorn/saltcorn#2453]
In the Settings - Site structure, it is possible to add dynamic menu items (Type: Dynamic).

Problem: how to display different menu items to different users depending on their user.id or custom field user.category?
The "Include formula" field does not take a value owner==user.id and it gives the error "Expression is too complicated"

**Comment by zmtk-josecardoso** [comment]
Hello. I'm looking for an way to do this also.
@castoutside , did you found a way?

**Comment by castoutside** [comment]
> Hello. I'm looking for an way to do this also.
> @castoutside , did you found a way?

No, unfortunately I don't have enough experience to figure out how to do this 🤷🏻‍♂️
